pkg/lib/operatorstatus/monitor.go

```diff
@@ -129,9 +129,18 @@ func (m *monitor) Run(stopCh <-chan struct{}) {
 	m.logger.Infof("initializing clusteroperator resource(s) for %s", m.names)
 
 	for _, name := range m.names {
-		if err := m.init(name); err != nil {
-			m.logger.Errorf("initialization error - %v", err)
-			break
+		for {
```

Comment:
As I read the code, I thought we might want to consider running this code in parallel for each "name", but I suspect that there isn't much of a performance gain given that:

1. I think the length of `m.names` is 3 or so entries.
2. Even if we ran this in parallel, I suspect that all three would fail if the controller was unable to initialize, so we'd just have 3 concurrent waits.

Reply:
I'm not sure it's worth it.

Reply:
Presumably, if the first one fails, then it makes sense to push off trying subsequent names until the first one succeeds. This puts less pressure on the API server.